### DISCUSSION TITLE
Getting `Unmatched Route` when navigating to /a/b/1

### DISCUSSION THREAD

**Discussion by khushal87** [expo/router#645] [General]
I have a folder structure in the app directory:
<img width="397" alt="Screenshot 2023-06-08 at 10 55 35" src="https://github.com/expo/router/assets/39884168/b0e05d25-ee87-498e-ae06-e409f7fa0352">
When I navigate to `/a/b/1` from my index.js using `router.push("/a/b/1")`. I am getting:
<img height="600px" width="300px" src="https://github.com/expo/router/assets/39884168/09175569-dd9a-4c31-a704-350180d7ade9"/>

What might be the cause of this?

**Comment by marklawlor** [comment]
Please provide a reproduction repo, there is not enough information provided and there is most likely another issue causing this, such as `[id].tsx` not having a default expo.

**Reply by khushal87** [reply]
Yes, you were right. I forgot to add the export default to the component. Adding it solved the issue.
